### DISCUSSION TITLE
Warning: uc_driver update needed || Chromedriver 121.0.6167.85

### DISCUSSION THREAD

**Discussion by Moiseosanchez** [ultrafunkamsterdam/undetected-chromedriver#1737] [Q&A]
```
Warning: uc_driver update needed. Getting it now:

*** chromedriver to download = 121.0.6167.85 (Latest Stable)

Downloading chromedriver-win64.zip from:
https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/121.0.6167.85/win64/chromedriver-win64.zip ...
```

I believe it's necessary to wait for the module to be updated, right? We can't do anything ourselves if the UC_driver is not updated by the developer, can we?

Thanks for this impressive project. 

Cheers.

**Comment by mdmintz** [comment]
That looks like you're using undetected-chromedriver via **seleniumbase**. seleniumbase displays those details when chromedriver is downloaded and patched. If your driver version doesn't match your browser version, seleniumbase downloads a new driver so that the versions match. Downloading and patching a driver should only take a few seconds (based on your Internet connection speed).

**Reply by Moiseosanchez** [reply]
Yes, it should it but I don't know why I download chrome-driver -from official site- and extract it into the folder and I have this message again and again... Any ideas?

**Reply by mdmintz** [reply]
It saves the driver to the `seleniumbase/drivers/` folder. Unless you delete the driver, or get a new version of Chrome, it shouldn't be downloading a new driver. Are you using the latest version of seleniumbase? What version do you have?

**Reply by dadashvespek** [reply]
I have the same issue on headless linux, on windows its fine but on linux every single time the driver gets initiated, I get this message

**Reply by mdmintz** [reply]
@dadashvespek Are you on the latest version? If not, upgrade. Otherwise, there may be a permissions issue.

**Reply by dadashvespek** [reply]
@mdmintz I am on the latest version. Not sure what the issue was but adding a specific driver version (eg: driver_version="112") fixed the issue!

**Reply by gitgetgotguts** [reply]
how did you add a new driver_version 
I updated SB but still get this warning everytime I start a new driver
